app/SQL/install.sql.sqlite.php

```diff
@@ -105,4 +105,12 @@
 		VALUES(:url, 1, :name, :website, :description, 86400);'
 );
 
-define('SQL_DROP_TABLES', 'DROP TABLE IF EXISTS `entrytag`, `tag`, `entrytmp`, `entry`, `feed`, `category`');
```

Comment:
Because SQLite does not support dropping multiple tables at once. This code never worked. 
But in my new PR, I have made all these calls uniform for the 3 databases.